### PR TITLE
Guided tour fixups

### LINKED ISSUE #1573: Guided tour overlay does not always go away when clicking "new grain" on IE
Tested on IE11

Steps:
1. Make sure you're logged in with an empty account.
2. Refresh to make sure you have a clean state.
3. Install an app that isn't already installed.
4. At the app details page you'll see the call-out of the "new grain" row. Click on the row to create a new grain.

Expected result: Callout overlay goes away when clicking "new grain".

Actual result:

screenshot from 2016-02-22 18-02-59

Note that clicking anywhere at this point causes the overlay to go away.

### LINKED ISSUE #1653: Highlighted box for guided tour (grains) doesn't dismiss itself
After clicking on "create new board" and having the Wekan UI load, the highlighted box from the guided tour still displayed. User paused for a moment then clicked on "got it" to dismiss it. Seems odd & unexpected for it to not go away after clicking on "create new board."

The guided tour should disappear after the user has taken the action it suggested to take, in this case "create new board."

(I spotted this while watching our most recent user test video. The cool thing about having screenshots of the demo is that it displays the timer too!)

image

image

image

### PR DESCRIPTION
**Pull request by paulproteus**
This pull request:

- Fixes  #1573 - guided tour dismissal failure on IE11

- Fixes #1653 - guided tour transparency failure on Firefox

I'm hoping for a speedy review & merge. Props to @zarvox for helping me realize that `Template.instance().data` is like `props`. :)

### CODE REVIEW

**Comment by neynah** [comment]
This is awesome, thanks for this fix!

**Comment by jparyani** [comment]
Garply, retest this please

File: shell/client/styles/shell.scss [hunk 1]
```diff
@@ -126,6 +126,12 @@ button.revoke-token, button.revoke-access {
   background-color: $default-content-background-color;
   overflow: auto;
 
+  // intro.js needs this element to have some numeric z-index, so that
+  // it knows to add the introjs-fixParent class to it. We use 0 because
+  // that is the value Chrome gives this element anyway, without specifying
+  // it explicitly.
+  z-index: 0;
```
Comment (kentonv) [inline shell/client/styles/shell.scss]: This broke first-time signup. Fixed in: aff63195b15cc9ae27bdd44b1d54e2a346563c31

Comment (paulproteus) [inline shell/client/styles/shell.scss]: Wow, thanks for catching that, Kenton.

**Pull request merged**